packages/node-integration-tests/suites/tracing/prisma-orm/scenario.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import { PrismaClient } from '@prisma/client';
+import * as Sentry from '@sentry/node';
+import * as Tracing from '@sentry/tracing';
+import { randomBytes } from 'crypto';
+
+const client = new PrismaClient();
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  release: '1.0',
+  tracesSampleRate: 1.0,
+  // @ts-ignore: We can't determine the type of the client from the integration
+  integrations: [new Tracing.Integrations.Prisma({ client })],
```

Comment:
Doesn't this `ts-ignore` mean that this will happen to any user? That kinda feels like bad UX. Could we change the client type to something `unknown`, validate it as a `PrismaClient` and throw an error if it isn't?

```ts
declare function isValidPrismaClient(possibleClient: unknown): possibleClient is PrismaClient;

public constructor(options: { client?: unknown } = {}) {
  if (isValidPrismaClient(options.client)) {
      this._client = options.client;
  } else {
    // TODO: Make a better logging message
    logger.warn(`Incorrect prisma client provided. Provided client: ${JSON.stringify(client)}`);
  }
}
```

Reply:
Yes, that's right, updated 👍